eng/ci/official-build.yml

```diff
@@ -12,6 +12,12 @@ trigger:
     include:
     - dev
 
+parameters:
+  - name: runEndToEndTests
+    type: boolean
+    default: true
+    displayName: "Run end-to-end tests"
```

Comment:
Why do we want to run e2e tests on official build for times other than scheduled builds?
It is probably good to run them after functional or code changes merged to main to see if the break some e2e flow.